blocks/contacts/__list/contacts__list.css

```diff
@@ -0,0 +1,29 @@
+.contacts__list{
+  margin: 0;
+  padding: 0;
+  display:flex;
+  flex-direction: row;
+  width: 74%;
+  justify-content: space-around;
+  gap: 70px;
+}
+
+@media (max-width: 660px){
+  .contacts__list{
+    flex-wrap: wrap;
```

Comment:
Could you please relocate the line `flex-wrap: wrap;` so that it applies to all screen sizes? This adjustment will fix the current horizontal scroll issue for screen sizes between 660px and 1200px.

![Screenshot 2023-11-16 at 5 58 19 PM](https://github.com/p-foundation/pink-app/assets/76534205/ce81e876-cecf-4114-a8d3-68ede03a4b1b)

Reply:
@naumch1k I have  relocated the line `flex-wrap: wrap;`